x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlAuthenticatorTests.java

```diff
@@ -108,97 +85,20 @@
 import static org.opensaml.saml.saml2.core.SubjectConfirmation.METHOD_ATTRIB_NAME;
 import static org.opensaml.saml.saml2.core.SubjectConfirmation.METHOD_BEARER;
 
-public class SamlAuthenticatorTests extends SamlTestCase {
+public class SamlAuthenticatorTests extends SamlResponseHandlerTests {
```

Comment:
Apologies for the merge conflicts that the SamlAuthenticatorTests refactoring causes here

Reply:
No worries. Wasn't too bad.